rpc/src/v1/helpers/dispatch.rs

```diff
@@ -474,7 +474,7 @@ pub fn execute<D: Dispatcher + 'static>(
 					.map(ConfirmationResponse::SignTransaction)
 				).boxed()
 		},
-		ConfirmationPayload::Signature(address, mut data) => {
+		ConfirmationPayload::PostSignTransaction(address, mut data) => {
```

Comment:
It doesn't necessarily need to be a transaction here. It's just a raw data to sign.